infra/mpc-recovery-dev/main.tf

```diff
@@ -64,12 +64,6 @@ resource "google_service_account_iam_binding" "serivce-account-iam" {
   ]
 }
 
-resource "google_project_iam_member" "service-account-datastore-user" {
```

Comment:
We don't have a dev environment for this service, but we should still keep this one for future reference just in case.